packages/core-app-api/api-report.md

```diff
@@ -225,7 +223,18 @@ export type AppOptions = {
   __experimentalTranslations?: {
     defaultLanguage?: string;
     availableLanguages?: string[];
-    resources?: Array<TranslationMessages | TranslationResource>;
+    resources?: Array<
```

Comment:
Figured it was unfortunate to force upgrades to TS 5.0 for everyone so inlined these instead, even if it wouldn't been within policy